test/qa/conftest.py

```diff
@@ -64,16 +71,58 @@ def print_to_string(*args, **kwargs):
 
 
 _original_print = print
+
+
 def _print_with_timestamp(*args, **kwargs):
     # Get the current time and format it
-    timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    timestamp = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
     # Prepend the timestamp to the original print arguments
     _original_print(timestamp, *args, **kwargs)
     logging.log(data=print_to_string(timestamp, *args, **kwargs))
 
 
+def _is_installed_as(package_type: str) -> bool:
+    """
+    Function to verify if installed nordvpn package is installed as deb or snap.
+
+    :param package_type:    The type of the package to check (DEB or SNAP).
+    :returns:               True if the package is installed, False otherwise.
+    """
+    return os.environ.get("NORDVPN_TYPE") == package_type
```

Comment:
`"NORDVPN_TYPE"` is repeated multiple times, should be put in constant to have single place for change